### PR TITLE
DEV-3891-add(test) : frontback test and solution

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by mohamedLazyBob — APPROVED** [review]

File: tests/frontback_test/main.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		"D", // D ---> CDE
+		"AB", // AB ---> \n
+		"a", // a ---> ab
+		"z", // z ---> yz
+		"HELLO WORLD", // \n
+		" ",// \n
+	}
+
+	for _, s := range table {
+		challenge.Program("frontback", strings.Fields(s)...)
```
Comment (MSilva95) [inline tests/frontback_test/main.go:22]: this is not testing, make the name of the folder in public and go-tests the same, and fix the naming in the func call as well accordingly

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/frontback_test/main.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"strings"
+	"github.com/01-edu/go-tests/lib/challenge"
+)
+
+func main() {
+	table := []string{
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply tests/frontback_test/main.go:9]: added

File: tests/frontback_test/main.go (hunk 1 above)
Comment (Hamzaelkhatri) [reply tests/frontback_test/main.go:27]: I change the name of the folder on the solution side, I guess the name is the same for the other part (subject & test)

**Review by MSilva95 — APPROVED** [review]

**Review by tiagocollot — APPROVED** [review]
LGTM! ⚡

**Pull request merged**